packages/block-library/src/navigation/index.php

```diff
@@ -270,31 +270,31 @@ function register_block_core_navigation() {
 		'core/navigation',
 		array(
 			'attributes' => array(
-				'className'          => array(
+				'className'             => array(
 					'type' => 'string',
 				),
-				'textColor'          => array(
+				'textColor'             => array(
 					'type' => 'string',
 				),
-				'rgbTextColor'       => array(
+				'customTextColor'       => array(
 					'type' => 'string',
 				),
-				'backgroundColor'    => array(
+				'backgroundColor'       => array(
 					'type' => 'string',
 				),
-				'rgbBackgroundColor' => array(
+				'customBackgroundColor' => array(
 					'type' => 'string',
 				),
-				'fontSize'           => array(
+				'fontSize'              => array(
 					'type' => 'string',
 				),
-				'customFontSize'     => array(
+				'customFontSize'        => array(
 					'type' => 'number',
 				),
-				'itemsJustification' => array(
+				'itemsJustification'    => array(
 					'type' => 'string',
 				),
-				'showSubmenuIcon'    => array(
+				'showSubmenuIcon'       => array(
 					'type'    => 'boolean',
 					'default' => true,
 				),
```

Comment:
Do we need a Block deprecation in place here? Not sure how it works when the attributes are in PHP land.

Reply:
We likely do need to move data from `rgb…` attr to `custom…` attr but only if the named color variant is not present. If the named color is present, `custom…` should be undefined

Reply:
https://github.com/WordPress/gutenberg/pull/20022/commits/054406cce5ca5cb04c2da2dffa9a964d9202c2cb handles the deprecated attributes as Marek mentioned above.

Reply:
- editor block deprecation: check
- backend compatibility for old instances: check